### PR TITLE
Update bg.po

### PR DESCRIPTION
**Pull request by NikoVP**
Update to latest master plus few small adjustments for better sound.

### CODE REVIEW

File: Translations/bg.po [hunk 1]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1527
+#: Source/diablo.cpp:1561
 msgid "Spellbook"
 msgstr "Книга за Заклинания"
 
-#: Source/diablo.cpp:1528
+#: Source/diablo.cpp:1562
 msgid "Open Spellbook."
-msgstr "Отваряне на книга с заклинания"
+msgstr "Отваряне на книга с заклинания. "
```
Comment (AJenbo) [inline Translations/bg.po:1351]: there are extra spaces t the en of a few of the lines

File: Translations/bg.po [hunk 2]
```diff
@@ -5596,113 +5607,109 @@ msgctxt "monster"
...
+#: Source/monster.cpp:4694 Source/monster.cpp:4704
+msgid " Fire"
 msgstr "Огън "
 
-#: Source/monster.cpp:4672 Source/monster.cpp:4684
-msgid "Lightning "
-msgstr "Мълния "
+#: Source/monster.cpp:4696 Source/monster.cpp:4706
+msgid " Lightning"
+msgstr "Мълния"
```
Comment (AJenbo) [inline Translations/bg.po:5652]: missing start space

File: Translations/bg.po [hunk 3]
```diff
@@ -5596,113 +5607,109 @@ msgctxt "monster"
...
 
-#: Source/monster.cpp:4678
-msgid "Immune: "
+#: Source/monster.cpp:4700
+msgid "Immune:"
 msgstr "Имунитет: "
```
Comment (AJenbo) [inline Translations/bg.po:5656]: remove space

File: Translations/bg.po (hunk 2 above)
Comment (NikoVP) [reply Translations/bg.po:5652]:
```suggestion
msgstr " Мълния"
```

**Comment by NikoVP** [comment]
Ok, Fixed the suggestions. I'll have another go at it later in the evening, so please hold on merging.

File: Translations/bg.po [hunk 4]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1421
+#: Source/diablo.cpp:1455
 msgid "Quick spell {}"
 msgstr "Бързо заклинание {}"
 
-#: Source/diablo.cpp:1422
+#: Source/diablo.cpp:1456
 msgid "Hotkey for skill or spell."
-msgstr "Клавиш за умение или заклинание"
+msgstr "Клавиш за умение или заклинание. "
```
Comment (AJenbo) [inline Translations/bg.po:1259]:
```suggestion
msgstr "Клавиш за умение или заклинание."
```

File: Translations/bg.po [hunk 5]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1405
+#: Source/diablo.cpp:1439
 msgid "Belt item {}"
 msgstr "Предмет от колана {}"
 
-#: Source/diablo.cpp:1406
+#: Source/diablo.cpp:1440
 msgid "Use Belt item."
-msgstr "Използване на предмет от колана"
+msgstr "Използване на предмет от колана "
```
Comment (AJenbo) [inline Translations/bg.po:1251]:
```suggestion
msgstr "Използване на предмет от колана."
```

File: Translations/bg.po [hunk 6]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1440
+#: Source/diablo.cpp:1474
 msgid "Speedbook"
 msgstr "Бързи Заклинания"
 
-#: Source/diablo.cpp:1441
+#: Source/diablo.cpp:1475
 msgid "Open Speedbook."
-msgstr "Отваряне на листа с бърз достъп до заклинанния"
+msgstr "Отваряне на листа с бърз достъп до заклинанния. "
```
Comment (AJenbo) [inline Translations/bg.po:1267]:
```suggestion
msgstr "Отваряне на листа с бърз достъп до заклинанния."
```

File: Translations/bg.po [hunk 7]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1495
+#: Source/diablo.cpp:1529
 msgid "Toggles if automap is displayed."
 msgstr "Включва показването на авто-карта."
 
-#: Source/diablo.cpp:1504
+#: Source/diablo.cpp:1538
 msgid "Open Inventory screen."
-msgstr "Отваряне на инвентар"
+msgstr "Отваряне на инвентар. "
```
Comment (AJenbo) [inline Translations/bg.po:1327]:
```suggestion
msgstr "Отваряне на инвентар."
```

File: Translations/bg.po [hunk 8]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1511
+#: Source/diablo.cpp:1545
 msgid "Character"
 msgstr "Герой"
 
-#: Source/diablo.cpp:1512
+#: Source/diablo.cpp:1546
 msgid "Open Character screen."
-msgstr "Отваряне на прозорец с информация за персонажа"
+msgstr "Отваряне на прозорец с информация за персонажа. "
```
Comment (AJenbo) [inline Translations/bg.po:1335]:
```suggestion
msgstr "Отваряне на прозорец с информация за персонажа."
```

File: Translations/bg.po [hunk 9]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1519
+#: Source/diablo.cpp:1553
 msgid "Quest log"
 msgstr "Задачи"
 
-#: Source/diablo.cpp:1520
+#: Source/diablo.cpp:1554
 msgid "Open Quest log."
-msgstr "Q: Отваряне на Приключенски дневник"
+msgstr "Q: Отваряне на Приключенски дневник. "
```
Comment (AJenbo) [inline Translations/bg.po:1343]:
```suggestion
msgstr "Q: Отваряне на Приключенски дневник."
```

File: Translations/bg.po [hunk 10]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1546
+#: Source/diablo.cpp:1580
 msgid "Hide Info Screens"
 msgstr "Скриване Инфо Екрани"
 
-#: Source/diablo.cpp:1547
+#: Source/diablo.cpp:1581
 msgid "Hide all info screens."
-msgstr "Скриване на всички информационни екрани"
+msgstr "Скриване на всички информационни екрани. "
```
Comment (AJenbo) [inline Translations/bg.po:1367]:
```suggestion
msgstr "Скриване на всички информационни екрани."
```

File: Translations/bg.po [hunk 11]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1566
+#: Source/diablo.cpp:1601
 msgid "Zoom"
 msgstr "Мащабиране"
 
-#: Source/diablo.cpp:1567
+#: Source/diablo.cpp:1602
 msgid "Zoom Game Screen."
-msgstr "Мащабиране на екрана"
+msgstr "Мащабиране на екрана. "
```
Comment (AJenbo) [inline Translations/bg.po:1375]:
```suggestion
msgstr "Мащабиране на екрана."
```

File: Translations/bg.po [hunk 12]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1583
+#: Source/diablo.cpp:1618
 msgid "Decrease Gamma"
 msgstr "Намаляване Гама"
 
-#: Source/diablo.cpp:1584
+#: Source/diablo.cpp:1619
 msgid "Reduce screen brightness."
-msgstr "Намаляване яркоста на екрана"
+msgstr "Намаляване яркоста на екрана. "
```
Comment (AJenbo) [inline Translations/bg.po:1391]:
```suggestion
msgstr "Намаляване яркоста на екрана."
```

File: Translations/bg.po [hunk 13]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1591
+#: Source/diablo.cpp:1626
 msgid "Increase Gamma"
 msgstr "Увеличаване Гама"
 
-#: Source/diablo.cpp:1592
+#: Source/diablo.cpp:1627
 msgid "Increase screen brightness."
-msgstr "Увеличаване яркостта на екрана"
+msgstr "Увеличаване яркостта на екрана. "
```
Comment (AJenbo) [inline Translations/bg.po:1399]:
```suggestion
msgstr "Увеличаване яркостта на екрана."
```

File: Translations/bg.po [hunk 14]
```diff
@@ -1181,223 +1210,231 @@ msgstr ""
...
-#: Source/diablo.cpp:1599
+#: Source/diablo.cpp:1634
 msgid "Help"
 msgstr "Помощ"
 
-#: Source/diablo.cpp:1600
+#: Source/diablo.cpp:1635
 msgid "Open Help Screen."
-msgstr "Помощен диалог"
+msgstr "Помощен диалог. "
```
Comment (AJenbo) [inline Translations/bg.po:1407]:
```suggestion
msgstr "Помощен диалог."
```

File: Translations/bg.po [hunk 15]
```diff
@@ -1423,22 +1460,22 @@ msgid "Crypt"
...
+#: Source/discord/discord.cpp:90
 msgid "Lv {} {}"
 msgstr "Нв {} {}"
 
 #. TRANSLATORS: Discord state i.e. "Nightmare difficulty"
-#: Source/discord/discord.cpp:100
+#: Source/discord/discord.cpp:102
 msgid "{} difficulty"
-msgstr "трудност {} "
+msgstr "{} трудност "
```
Comment (AJenbo) [inline Translations/bg.po:1475]:
```suggestion
msgstr "{} трудност"
```

File: Translations/bg.po [hunk 16]
```diff
@@ -1448,262 +1485,262 @@ msgstr "Обратна връзка"
 
 #: Source/dvlnet/tcp_client.cpp:65
 msgid "Unable to connect"
-msgstr "Свързването невъзможно "
+msgstr "Свързването e невъзможно "
```
Comment (AJenbo) [inline Translations/bg.po:1488]:
```suggestion
msgstr "Свързването e невъзможно"
```

File: Translations/bg.po [hunk 17]
```diff
@@ -5596,113 +5607,109 @@ msgctxt "monster"
...
+#: Source/monster.cpp:4690
+msgid "Resists:"
 msgstr "Устойчивост: "
 
-#: Source/monster.cpp:4668 Source/monster.cpp:4680
-msgid "Magic "
-msgstr "Магия "
+#: Source/monster.cpp:4692 Source/monster.cpp:4702
+msgid " Magic"
+msgstr "Магия"
```
Comment (AJenbo) [inline Translations/bg.po:5644]:
```suggestion
msgstr " Магия"
```

File: Translations/bg.po [hunk 18]
```diff
@@ -5596,113 +5607,109 @@ msgctxt "monster"
...
-#: Source/monster.cpp:4662
+#: Source/monster.cpp:4687
 msgid "No magic resistance"
 msgstr "Няма магическа устойчивост"
 
-#: Source/monster.cpp:4666
-msgid "Resists: "
+#: Source/monster.cpp:4690
+msgid "Resists:"
 msgstr "Устойчивост: "
```
Comment (AJenbo) [inline Translations/bg.po:5640]:
```suggestion
msgstr "Устойчивост:"
```

File: Translations/bg.po [hunk 19]
```diff
@@ -5596,113 +5607,109 @@ msgctxt "monster"
...
-msgstr "Магия "
+#: Source/monster.cpp:4692 Source/monster.cpp:4702
+msgid " Magic"
+msgstr "Магия"
 
-#: Source/monster.cpp:4670 Source/monster.cpp:4682
-msgid "Fire "
+#: Source/monster.cpp:4694 Source/monster.cpp:4704
+msgid " Fire"
 msgstr "Огън "
```
Comment (AJenbo) [inline Translations/bg.po:5648]:
```suggestion
msgstr " Огън"
```

**Comment by NikoVP** [comment]
Done. I have no Idea how these spaces came to be. hope this resolves it. I will do more updates in the future , some of the dialogs are still quite stiff. And I see mistakes better ingame. Cheers!

**Pull request merged**